easybuild/easyconfigs/f/FastEpistasis/FastEpistasis-2.07-foss-2018a.eb

```diff
@@ -0,0 +1,38 @@
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2021 University of Geneva Switzerland
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'FastEpistasis'
+
+version = '2.07'
+
+homepage = 'https://gitlab.sib.swiss/tschuepb/FastEpistasis'
+
+description = '''
+   FastEpistasis, a software tool capable of computing tests of epistasis for a large number of SNP pairs, is an 
+   efficient parallel extension to the PLINK epistasis module.'''
+
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://gitlab.sib.swiss/tschuepb/FastEpistasis/uploads/864695cfa08720b46dd1b437faca3fff/']
+sources = ['%(name)s-%(version)s.tar.bz2']
+checksums = ['3751461a0c410749ce77b07b957f7e9c4cbfb18aa4003b5aab3fc52db10b5f33']
+
+configopts = '-DCOMPILE_MPI_EPISTASIS=TRUE'
+
+dependencies = [
+    ('CMake', '3.10.2'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/smp%(name)s'],
```

Comment:
Can we add a simple sanity check command for this too? Like using `--help`?

Reply:
Hi, I tried but the binary returns 1 when using `-h`.

Reply:
One way of working around that is to pipe the output to `grep`, and check for a particular pattern. That way, you're at least checking that the binary runs and produces meaningful output.

Reply:
done